### PR TITLE
fix(templates, crm grid, grid batch editing): Fixing issues in the tw…

### PR DESCRIPTION
**Pull request by valadzhov**
…o templates.

### CODE REVIEW

File: packages/igx-templates/igx-ts/custom-templates/crm-grid/files/src/app/__path__/__filePrefix__.component.html [hunk 1]
```diff
@@ -32,16 +32,16 @@
 							</span>
 						</div>
 						<div class="caseSensitiveButton">
-							<button igxButton="icon" igxRipple igxRippleCentered="true" (click)="updateSearch()"
+							<button igxButton="icon" igxRipple [igxRippleCentered]="true" (click)="updateSearch()"
```
Comment (dafo) [inline packages/igx-templates/igx-ts/custom-templates/crm-grid/files/src/app/__path__/__filePrefix__.component.html:35]: No need for square brackets here and below as well as the value is set in the template.

File: packages/igx-templates/igx-ts/custom-templates/crm-grid/files/src/app/__path__/__filePrefix__.component.html [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <div class="grid__wrapper">
-	<igx-grid #grid1 [data]="localData" [height]="'430px'" [width]="'1180px'" [rowSelection]="'multiple'"
+	<igx-grid #grid1  igxPreventDocumentScroll id="grid1" [data]="localData" [height]="'430px'" [width]="'1180px'" [rowSelection]="selectionMode"
```
Comment (dafo) [inline packages/igx-templates/igx-ts/custom-templates/crm-grid/files/src/app/__path__/__filePrefix__.component.html:2]: Changing the rowSelection in this case does not hurt, however - it is a bit obsolete as we are just setting it to 'multiple' in the .ts file. So, it is up to you to decide if the change is really needed.

Comment (valadzhov) [reply packages/igx-templates/igx-ts/custom-templates/crm-grid/files/src/app/__path__/__filePrefix__.component.html:2]: Changed to the suggested.

File: packages/igx-templates/igx-ts/custom-templates/crm-grid/files/src/app/__path__/__filePrefix__.component.html (hunk 1 above)
Comment (valadzhov) [reply packages/igx-templates/igx-ts/custom-templates/crm-grid/files/src/app/__path__/__filePrefix__.component.html:35]: Changed to the suggested.

**Review by dafo — APPROVED** [review]

**Review by Lipata — APPROVED** [review]

**Pull request merged**